pinot-common/src/main/java/org/apache/pinot/common/utils/request/RequestUtils.java

```diff
@@ -301,6 +304,43 @@ public static Expression getLiteralExpression(Object object) {
     return RequestUtils.getLiteralExpression(object.toString());
   }
 
+  public static Function getFunction(String canonicalName, List<Expression> operands) {
+    Function function = new Function(canonicalName);
+    function.setOperands(operands);
+    return function;
+  }
+
+  public static Function getFunction(String canonicalName, Expression operand) {
+    // NOTE: Create an ArrayList because we might need to modify the list later
+    List<Expression> operands = new ArrayList<>(1);
+    operands.add(operand);
+    return getFunction(canonicalName, operands);
+  }
+
+  public static Function getFunction(String canonicalName, Expression... operands) {
+    // NOTE: Create an ArrayList because we might need to modify the list later
+    return getFunction(canonicalName, new ArrayList<>(Arrays.asList(operands)));
```

Comment:
just loop over operands and add to the new list, w/o calling Arrays.asList() to avoid creating a temp list.

Reply:
I actually considered that, but not sure if that is really faster. The trade-off is one extra array copy vs a lot of extra method invocations and a for loop. Seems IDE doesn't like the for loop, thus I decided to use the IDE suggested way